msal/src/test/java/com/microsoft/identity/client/e2e/tests/AcquireTokenAbstractTest.java

```diff
@@ -82,39 +82,43 @@ public void performInteractiveAcquireTokenCall(final String username, final Stri
         flushScheduler();
     }
 
-    public void performSilentAcquireTokenCall(final String[] scopes) {
+    public void performSilentAcquireTokenCall(final String[] scopes,
+                                              final boolean expectingResultFromCache) {
```

Comment:
From the name, it's not immediately obvious what `expectingResultFromCache` is, or when it's `true`/`false`

Can we add some javadoc?

Reply:
Or just comment/explainer for this flag. Don't necessarily need to javadoc tests

Reply:
(Didn't check file name before commenting; realize now that this is a test file)

Reply:
Addressed here: https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/1048/commits/f09a7afc15134b05fd6c23615308bfa4ea53dbdf